lib/job.js

```diff
@@ -245,10 +247,15 @@ Job.prototype = {
         }
         
         next(exitCode, std.out, std.err);
-      });
+      };
+
+      proc.on('exit', done);
 
-      proc.on('error', function() {
-        // prevent node from throwing an error
```

Comment:
From https://nodejs.org/api/child_process.html#child_process_event_error

> Note that the exit-event **may or may not fire after an error has occurred**. If you are listening on both events to fire a function, remember to guard against calling your function twice.

So we do need an error handler.